packages/code-du-travail-frontend/src/web-components/tooltip.js

```diff
@@ -58,14 +59,14 @@ class WebComponentsTooltip extends LitElement {
         }
       }
 
-      .tooltip .content.visible {
+      .content.visible {
         display: block;
       }
     `;
   }
 
   render() {
-    return html`<div
+    return html` <div
```

Comment:
```suggestion
    return html`<div
```

Reply:
It's automated formatted by prettier. Should we change the prettier rules?

Reply:
ok, I was not sure it was intentionnal. we leave it like that